Won't compile with node v7.5

`../src/tag.cc:239: error: ‘Loop’ was not declared in this scope`

`node::Loop()` no longer exists in node latest. It's purpose was to support isolates in node.js. The isolate support project has been canceled in node core and as a result `node::loop()` no longer exists in node.h

The fix is a simple one liner. Just replace `Loop()` on line 239 of src/tag.cc with `uv_default_loop()` and this library will compile with node v7.5

--- a/src/tag.cc
+++ b/src/tag.cc
@@ -236,7 +236,7 @@ v8::Handle<v8::Value> Tag::AsyncTag(const v8::Arguments &args) {
     baton->callback = Persistent<Function>::New(callback);
     baton->error = 0;
 
-    uv_queue_work(Loop(), &baton->request, Tag::AsyncTagRead, Tag::AsyncTagReadAfter);
+    uv_queue_work(uv_default_loop(), &baton->request, Tag::AsyncTagRead, Tag::AsyncTagReadAfter);
 
     return Undefined();
 }

